cicd/pipelines/creating-applications-with-cicd-pipelines.adoc

```diff
@@ -63,13 +63,20 @@ include::modules/op-creating-webhooks.adoc[leveloffset=+1]
 
 include::modules/op-triggering-a-pipelinerun.adoc[leveloffset=+1]
 
+include::modules/op-enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects.adoc[leveloffset=+1]
+
+[role="_additional-resources_op-enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects"]
+.Additional resources
```

Comment:
Missing an ID, e.g.
```
[role="_additional-resources"]
[id="additional-resources_creating-applications-with-cicd-pipelines"]
== Additional resources
```
please see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies

Reply:
Oops. My mistake. Thanks.